jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -60,21 +60,21 @@ def _check_is_file(path):
 
 def _warn_if_not_found(app, file_lists):
     """
-    Take a list of labels and associated file lists and send a snackbar
-    message if the length of any list is 0.
+    Take a list of labels and associated file lists/strings and send a
+    snackbar message if any are empty.
     """
     found = []
     not_found = []
     for key in file_lists:
-        if len(file_lists[key]) == 0:
-            not_found.append(key)
-        else:
+        if file_lists[key]:
             found.append(key)
+        else:
+            not_found.append(key)
 
     if len(found) == 0:
         raise ValueError("No valid files found in specified directory")
     if len(not_found) > 0:
-        warn_msg = "Some files not found: {}".format(", ".join(not_found))
+        warn_msg = f"Some files not found: {', '.join(not_found)}"
         print(warn_msg)
```

Comment:
Not your diff but this looks like leftover debugging?
```suggestion
```

Reply:
Yeah, I was wondering why it should print _and_ broadcast. I'll accept the deletion since we had the same thought.